shrinkwrap-resolver/src/test/resources/custom-settings.xml

```diff
@@ -28,7 +28,7 @@
                 <repository>
                     <id>custom-maven-repo</id>
                     <name>Ibiblio Mirror</name>
```

Comment:
@sgilda I haven't noticed that you didn't rewrite the name of the repository. Could you do it to reflect the new custom repository?